internal/operator/subscription.go

```diff
@@ -25,13 +25,13 @@ func Subscriptions(catalogSource string, catalogSourceNamespace string) ([]Subsc
 
 	c, err := NewPackageServerClient()
 	if err != nil {
-		logger.Errorf("Error while creating new PackageServerClient: %s", err)
+		logger.Errorw("Error while creating new PackageServerClient", err)
```

Comment:
Errorf instead of Errorw? If Errorw need to fix format; logger.Errorw("Error while creating new PackageServerClient", "error", err)